### ISSUE TITLE
Exclude "think" from results

### ISSUE THREAD

**Issue by astanciu** [shoreditch-ops/artillery#360]
Sometimes it's necessary to use "think" to make a scenario wait a little, perhaps the backend is doing a background job. It would be nice to have an option to exclude the sum of all "think"'s from the result times.  WDYT?

**Comment by hassy** [comment]
Do you mean excluding the time spent in `think` from scenario duration reports?

**Comment by astanciu** [comment]
Yes, `think` time is client side, which isn't particularly interesting.
